src/service/components/pulseaudio.js

```diff
@@ -2,40 +2,43 @@
 //
 // SPDX-License-Identifier: GPL-2.0-or-later
 
-'use strict';
+import GIRepository from 'gi://GIRepository';
+import GLib from 'gi://GLib';
+import GObject from 'gi://GObject';
 
-const Tweener = imports.tweener.tweener;
-
-const GIRepository = imports.gi.GIRepository;
-const GLib = imports.gi.GLib;
-const GObject = imports.gi.GObject;
+import Config from '../../config.js';
 
-const Config = imports.config;
+const Tweener = imports.tweener.tweener;
 
 
-// Add gnome-shell's typelib dir to the search path
-const typelibDir = GLib.build_filenamev([Config.GNOME_SHELL_LIBDIR, 'gnome-shell']);
-GIRepository.Repository.prepend_search_path(typelibDir);
-GIRepository.Repository.prepend_library_path(typelibDir);
+let Gvc = null;
+try {
+    // Add gnome-shell's typelib dir to the search path
+    const typelibDir = GLib.build_filenamev([Config.GNOME_SHELL_LIBDIR, 'gnome-shell']);
+    GIRepository.Repository.prepend_search_path(typelibDir);
+    GIRepository.Repository.prepend_library_path(typelibDir);
 
-const Gvc = imports.gi.Gvc;
+    Gvc = (await import('gi://Gvc')).default;
+} catch (e) {}
```

Comment:
Is it possible that having this in a try/catch, combined with `await import()`, is causing failures to resolve imports when they're needed?